worlds_disabled/README.md

```diff
@@ -3,3 +3,18 @@
 This folder is for already merged worlds that are unmaintained and currently broken. If you are interested in fixing and
 stepping up as maintainer for any of these worlds, please review the [world maintainer](/docs/world%20maintainer.md)
 documentation.
+
+## Rationale for Disabled Worlds
+
+Listed below is the reason for each world being disabled. For a world to be considered for reactivation, all concerns
+should be addressed, however this requirement does not guarantee reactivation in and of itself. Depending on the age 
+of the disabled worlds, additional issues may need to be handled that were not present previously.
+
+
+### Ori and the Blind Forest (./oribf) 
+
+  * World fails tests if world was not `hidden`
+  * No client exists for world
+  * No maintainer for world
+  * Not listed in README.md
+  * Removes special rule in `testDefaultAllStateCanReachEverything`
```

Comment:
```suggestion
```

Reply:
This was just a change that occurred as a part of the commit to disable Ori. This would effectively fall under the first bullet.